### DISCUSSION TITLE
Using Time.timeScale and SceneManagement

### DISCUSSION THREAD

**Discussion by pastaread** [spatialsys/spatial-unity-sdk#18] [Help]
1. I've got this "The symbol 'UnityEngine.Time.set_timeScale' is banned in this project", how to pause/resume game without it?
2. After GameOver how to load current scene, "The symbol 'UnityEngine.SceneManagement.Scene.get_buildIndex' is banned in this project "

**Comment by AlanSpatial** [comment]
We currently don't support setting the timescale which makes pausing games tricky... But keep in mind that spatial is a multiplayer platform and you normally don't pause multiplayer games. We'd like to support setting timescale, but not in the immediate future.

We don't allow loading or re-loading scenes. So youll need to manage resetting your gamestate manually.
